zipkin-server/src/main/java/zipkin/server/ZipkinServer.java

```diff
@@ -21,11 +21,6 @@
 @SpringBootApplication
 @EnableZipkinServer
 public class ZipkinServer {
-  static {
```

Comment:
think maybe need to install the logging bridge (you can try hitting the dependencies endpoint on trace level when you have spans loaded)